app/javascript/dashboard/routes/dashboard/helpcenter/components/HelpCenterLayout.vue

```diff
@@ -224,7 +224,9 @@ export default {
           key: 'category',
           children: this.categories.map(category => ({
             id: category.id,
-            label: category.name,
+            label: category.icon
```

Comment:
Instead of this, it would be better to have the JSON response from the backend to include it as a `label` field

Updating the `app/views/public/api/v1/models/hc/_portal.json.jbuilder` should work

Reply:
@pranavrajs What do you think about this? Do we need a separate icon key? We are using it in the public portal.

Reply:
We can keep the icon key, just have a label from the backend, makes it instantly usable across the app

Reply:
Okay, do you mean to add a new key named label with value icon + name?

Reply:
Let's keep it as is for now, but we can make changes in the future. Thoughts? @pranavrajs

Reply:
Sounds good!